cases/management/commands/_enrich_utils.py

```
@@ -0,0 +1,228 @@
+        import likhit  # noqa: F401
+        from markitdown import MarkItDown
+
+        md = MarkItDown(enable_plugins=True)
+        result = md.convert(tmp_path)
+
+        if result and result.text_content and len(result.text_content.strip()) > 200:
+            return result.text_content.strip()
+
+        logger.warning("Likhit conversion produced insufficient content for %s", url)
+        return None
+    except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)